packages/2d/src/components/Layout.ts

```diff
@@ -157,18 +158,32 @@ export class Layout extends Node {
   @initial(null)
   @signal()
   public declare readonly fontFamily: SimpleSignal<string | null, this>;
-  @initial(null)
   @signal()
-  public declare readonly fontSize: SimpleSignal<number | null, this>;
+  public declare readonly fontSize: SimpleSignal<number, this>;
+
+  protected computeFontSize(): number {
+    this.requestLayoutUpdate();
+    return Number(this.styles.getPropertyValue('font-size').slice(0, -2));
+  }
+
   @initial(null)
   @signal()
   public declare readonly fontStyle: SimpleSignal<string | null, this>;
   @initial(null)
   @signal()
   public declare readonly fontWeight: SimpleSignal<number | null, this>;
-  @initial(null)
   @signal()
-  public declare readonly lineHeight: SimpleSignal<number | null, this>;
+  public declare readonly lineHeight: SimpleSignal<number, this>;
+
+  protected computeLineHeight(): number {
```

Comment:
There's already a way to make the line height depend on the font size in CSS. I believe that's how we should do that, similarly to how `width` can have different units.